aw_datastore/storages/__init__.py

```diff
@@ -0,0 +1,16 @@
+import logging as _logging
+logger = _logging.getLogger("aw.datastore.storages")  # type: _logging.Logger
+
+import appdirs as _appdirs
+import os as _os
+# TODO: This should probably be placed in a differently named folder, but will certainly break stuff
+data_dir = _appdirs.user_data_dir("aw-server", "activitywatch")  # type: str
```

Comment:
Good point, we should probably add a issue for this.

Related to that, i have wondered if it's worth moving aw_database into a seperate repository. It's a very low priority task but it's worth considering.